lib/content-services/src/lib/search/components/search-date-range-tabbed/search-date-range-tabbed.component.ts

```diff
@@ -60,12 +62,39 @@ export class SearchDateRangeTabbedComponent implements SearchWidget, OnInit {
     private value: { [key: string]: Partial<SearchDateRange> } = {};
     private queryMapByField: Map<string, string> = new Map<string, string>();
     private displayValueMapByField: Map<string, string> = new Map<string, string>();
+    private destroy$ = new Subject<void>();
 
     constructor(private translateService: TranslationService) {}
 
     ngOnInit(): void {
         this.fields = this.settings?.field.split(',').map((field) => field.trim());
         this.setDefaultDateFormatSettings();
+        this.context.populateFilters
+            .asObservable()
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((filtersQueries) => {
+                if (filtersQueries[this.id]) {
```

Comment:
Inside that subscribe you are getting value by id from that object many times. Can you instead put result of filtersQueries[this.id] to some const? Or use some rxjs operator to get only this one value in subscribe?